setup.cfg

```diff
@@ -3,5 +3,6 @@
 universal=1
 
 [bdist_rpm]
+# Due to changes in Python's ConfigParser, this only works in Python 3. For Python 2, remove one %
+release=1%%{?dist}
 requires=python-six
-build_requires=python-nose
```

Comment:
As far as I could tell, this was not actually needed for the build.  Which is handy, because this package doesn't exist on Centos stream 9